core/src/main/java/org/apache/iceberg/TableMetadata.java

```diff
@@ -274,6 +275,59 @@ public String toString() {
       Map<String, SnapshotRef> refs,
       List<StatisticsFile> statisticsFiles,
       List<MetadataUpdate> changes) {
+    this(
+        metadataFileLocation,
+        formatVersion,
+        uuid,
+        location,
+        lastSequenceNumber,
+        lastUpdatedMillis,
+        lastColumnId,
+        currentSchemaId,
+        schemas,
+        defaultSpecId,
+        specs,
+        lastAssignedPartitionId,
+        defaultSortOrderId,
+        sortOrders,
+        properties,
+        currentSnapshotId,
+        snapshots,
+        snapshotsSupplier,
+        snapshotLog,
+        previousFiles,
+        refs,
+        statisticsFiles,
+        changes,
+        null);
```

Comment:
this is mainly so that we don't have to update a bunch of tests that use this constructor signature